vortex-duckdb/src/convert/vector.rs

```diff
@@ -0,0 +1,317 @@
+/// Copied of duckdb-rs (https://github.com/duckdb/duckdb-rs/blob/main/crates/duckdb/src/vtab/arrow.rs)
```

Comment:
Any unit tests we can copy too? :)

Reply:
I think we should just move over to duckdb -> vortex directly so I would way there is less value in porting tests over